Sources/MediaNavigationBar.swift

```diff
@@ -48,7 +55,9 @@ protocol MediaNavigationBarDelegate {
         chromeButton.addTarget(self, action: #selector(toggleChromeCast), for: .touchUpInside)
         chromeButton.setImage(UIImage(named: "renderer"), for: .normal)
         chromeButton.tintColor = .white
+        chromeButton.widthAnchor.constraint(equalToConstant: buttonSize).isActive = true
         chromeButton.translatesAutoresizingMaskIntoConstraints = false
+        chromeButton.isEnabled = false // disable until a chromecast device is found
```

Comment:
that button should not be visible I assume when there is no chromecast device

Reply:
hmm would it not be just as okay to have the button but not have it selectable if a renderer isnt present?